src/test/java/org/apache/sysds/test/functions/federated/algorithms/FederatedLmPipeline.java

```diff
@@ -19,6 +19,7 @@
 
 package org.apache.sysds.test.functions.federated.algorithms;
 
+import com.sun.tools.javac.util.List;
```

Comment:
I think you have some automated imports that have to change priority.
Usually we don't import the com.sun libraries.

Reply:
import java.util.List;

Reply:
and to create an array in oneline use:

```java
int[] x = new int[] {1, 20, 3, 40, 0}
```